shared/wallets/wallet/settings/index.js

```diff
@@ -56,92 +56,98 @@ const HoverText = Styles.isMobile
-              color={Styles.globalColors.red}
+          <Kb.Box2 direction="vertical" style={styles.sectionLabel}>
+            <Kb.Text type="BodySmallSemibold">Display currency</Kb.Text>
+          </Kb.Box2>
+          <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.dropdownContainer} gap="tiny">
+            <Kb.Dropdown
+              disabled={props.currencyWaiting}
+              items={makeDropdownItems(props.currencies, props.currency)}
+              selected={makeDropdownItem(props.currency, false)}
+              onChanged={(node: React.Node) => {
+                // $ForceType doesn't understand key will be string
+                const selectedCode: Types.CurrencyCode = node.key
+                if (selectedCode !== props.currency.code && selectedCode !== headerKey) {
+                  props.onCurrencyChange(selectedCode)
+                }
+              }}
+              style={styles.dropdown}
             />
-            <Kb.Text
-              type="BodySemibold"
-              style={Styles.collapseStyles([styles.red, props.isDefault && styles.deleteOpacity])}
-              className={props.isDefault ? '' : 'hover-underline'}
-            >
-              Remove account
-            </Kb.Text>
-          </Kb.ClickableBox>
-          {props.isDefault && (
-            <Kb.Text style={styles.centerText} type="BodySmall">
-              You can’t remove your default account.
-            </Kb.Text>
-          )}
+            <Kb.SaveIndicator saving={props.currencyWaiting} minSavingTimeMs={300} savedTimeoutMs={2500} />
+          </Kb.Box2>
+          <Kb.Text type="BodySmall">The display currency appears:</Kb.Text>
+          <Kb.Text type="BodySmall">- near your Lumens balance</Kb.Text>
+          <Kb.Text type="BodySmall">- when sending or receiving Lumens</Kb.Text>
+          <Kb.Box2 direction="vertical" fullWidth={true} style={styles.removeContainer}>
+            <Kb.ClickableBox style={styles.remove} onClick={props.isDefault ? null : props.onDelete}>
+              <Kb.Icon
+                type="iconfont-trash"
+                style={Styles.collapseStyles([styles.rightMargin, props.isDefault && styles.deleteOpacity])}
+                color={Styles.globalColors.red}
+              />
+              <Kb.Text
+                type="BodySemibold"
+                style={Styles.collapseStyles([styles.red, props.isDefault && styles.deleteOpacity])}
+                className={props.isDefault ? '' : 'hover-underline'}
+              >
+                Remove account
+              </Kb.Text>
+            </Kb.ClickableBox>
+            {props.isDefault && (
+              <Kb.Text style={styles.centerText} type="BodySmall">
+                You can’t remove your default account.
+              </Kb.Text>
+            )}
+          </Kb.Box2>
         </Kb.Box2>
       </Kb.Box2>
-    </Kb.Box2>
+    </Kb.ScrollView>
```

Comment:
This page could use some space at the bottom (iPhone 5S)
<img width="377" alt="image" src="https://user-images.githubusercontent.com/11968340/46747602-bf779c00-cc7f-11e8-9a48-b1a9aa583271.png">